modules/core/src/main/java/org/apache/ignite/IgniteCluster.java

```diff
@@ -453,6 +453,21 @@
      */
     public void active(boolean active);
 
+    /**
+     * Checks Ignite grid in read-only mode or not.
+     *
+     * @return {@code true} if grid is in read-only mode and {@code false} If grid allows data modification operations.
+     */
+    public boolean readOnly();
```

Comment:
Do we need throwing an exception here ? What happens if the method is called when cluster is not activated ?

Reply:
```